docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -6,83 +6,153 @@ brokers etc. There are two kind of events:
-}
-```
-
-There are values starting with `$` e.g. `$endpointName`. These are called variables.
+option that have no validation at all.
+
+=== "JSON"
+
+    ```json
+    {
+      "version": "$version",
+      "timestamp": $timestamp,
+      "endpointName": "$endpointName",
+      "domainId": "$domainId",
+      "url": "$url",
+      "value": "$value"
+    }
+    ```
+
+=== "TEXT"
+
+    ```text
+    version: $version
+    timestamp: $timestamp
+    endpointName: $endpointName
+    domainId: $domainId
+    url: $url
+    value: $value
+    ```
+
+Note that there are values starting with `$` e.g. `$endpointName`. These are called variables.
 
 ## Variables
 
 Variables are predefined and allow for using some device context in a payload. When sending event, variables are replaced 
 with actual value e.g. `$endpointName` with `test-device-47193512895`. There are several variables that can be used, if
-variable does not exist e.g. `$notExisting` then it is sent as normal text. Available variables differs from context where
-they are used.
+variable does not exist e.g. `$notExisting` then it is sent as normal text.
+
+=== "Device Lifecycle Variables"
+
+      * `version` *(type: string)* - The version in which events are sent. Any changes will be reflected in it.
+      * `timestamp` *(type: long)* - Time of occurrence of the event in milliseconds.
+      * `endpointName` *(type: string)* - The device endpoint name given at creation.
+      * `domainId` *(type: string)* - The entire ID of the domain in which the device is located.
+      * `domainName` *(type: string)* - The name of the last subdomain without slashes.
+      * `subtype` *(type: string)* - Enumeration type of the event.
+      * `metadata` *(type: json|string)* - Additional details about the event in relation to its subtype.
+
+        !!! Note "Metadata format"
+            For the `JSON` content type, it is an object with possibly nested fields. Whereas for `TEXT` it is a simple string with fields separated by commas.
+
+=== "Device Telemetry Variables"
+
+      * `version` *(type: string)* - The version in which events are sent. Any changes will be reflected in it.
+      * `timestamp` *(type: long)* - Time of occurrence of the event in milliseconds.
+      * `endpointName` *(type: string)* - The device endpoint name given at creation.
+      * `domainId` *(type: string)* - The entire ID of the domain in which the device is located.
+      * `domainName` *(type: string)* - The name of the last subdomain without slashes.
+      * `url` *(type: string)* - LwM2M path (Resource Id) for which changes are occurring.
+      * `value` *(type: string|number)* - The value of the changed LwM2M resource. 
+
+        !!! Note "Value format"
+            Although it can be a number, in the case of `JSON` content type it must be enclosed in quotation marks.
+
+Available variables differs from context where they are used. See [headers](#headers) and [body](#body) for detailed information.
```

Comment:
Purpose first. Hence: "For detailed information, see [headers](#headers) and [body](#body).